auctions/english-auction/packages/app/src/systems/Sell/routes.tsx

```diff
@@ -6,7 +6,7 @@ import { Pages } from "~/types";
 
 export const sellRoutes = (
   <>
-    <Route path="/" element={<CreateAuctionPage />} />
+    <Route path="*" element={<CreateAuctionPage />} />
```

Comment:
It's a wildcard so if a user goes to any other route besides the ones defined then it will fall back on this route and load the sell/create auction page.